docs/commands.rst

```diff
@@ -10,5 +10,6 @@ Commands
    commands/run-data-managers
    commands/setup-data-libraries
    commands/shed-tools
+   commands/tool-deps
```

Comment:
Maybe install-tool-deps?

Reply:
If you could add sub-commands like show, install, remove, reinstall ... then this name would make sense imho. Do you plan to add those?

Reply:
I was thinking of having sub-commands but wanted feedback before implementing all of that :)

I'll take a crack at them tomorrow.

Reply:
Looks like bioblend provides no means of inspecting or uninstalling a tool dependency